### ISSUE TITLE
Do you have any idea how to make it working on mobile?

### ISSUE THREAD

**Issue by gustawdaniel** [Property404/its-a-unix-system#1]
The problem is that in js its quite hard to start mobile keyboard.

I was trying with `contenteditable` but without success. I mean, keyboard was visible but terminal didn't work.

Maybe hidden input with label on #terminal can work, but it would require to redirect keypress from this input to #terminal.

**Comment by Property404** [comment]
No, I looked into a bit before but haven't been able to figure it out
